metron-maven-archetypes/README.md

```diff
@@ -0,0 +1,56 @@
+# Metron Maven Archetypes
+
+These are [Apache Maven Archetypes](http://maven.apache.org/archetype/index.html) for use in creating extension components for the Apache Metron system
+Archetypes are helpful in creating maven based projects with the correct setup and dependency configurations for the target system, including providing sample implementations.
+
+### metron-maven-parser-extension-archetype
+This is an archetype for creating an Apache Metron Parser Extension
+
+#### Use
+
+Build and install the archetype
+```
+$ mvn install
+```
+
+Create a directory to host your extension code
+
+```
+$ mkdir ~/src/my-parser-extension
```

Comment:
Just a suggestion - the predefined extensions have directory names and structures like this:
```
* metron-parser-extensions
  * metron-parser-extension-<sensor name>-extension
    * metron-parser-<sensor name>
    * metron-parser-<sensor name>-assembly
    * metron-parser-<sensor name>-bundle
```

To give the reader context, I would consider changing `~/src/my-parser-extension` to `~/src/metron-parser-nice-extension`